resources/public/api/conf/1.0/schemas/retrieve_charitable_giving_response.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "title":"Charitable Givings",
+  "description":"Charitable Givings",
+  "type":"object",
+  "properties":{
+    "giftAidPayments":{
+      "description":"An object containing Gift Aid payments made to charities for the specified tax year.",
+      "type":"object",
+      "properties":{
+        "nonUkCharities":{
+          "description":"An object containing Gift Aid payments made to charities for the specified tax year.",
```

Comment:
'nonUkCharities' description is a different in the spec. You are missing the 'non-UK' bit after made to